recipes/powerlaw/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "powerlaw" %}
+{% set version = "1.4.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "7f9d9cb39ad23af8f6cae165d47020ee8b7fa58f380c356de91ae2625577699e"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - matplotlib
+    - mpmath
+    - numpy
+    - pip
+    - python
+    - scipy
+  run:
+    - matplotlib
+    - mpmath
+    - numpy
+    - python
+    - scipy
+test:
+  imports:
+    - powerlaw
+
+about:
+  home: "http://www.github.com/jeffalstott/powerlaw"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "license.txt" 
```

Comment:
You can list the `license_file` as `powerlaw.py` since the license is included there

Reply:
powerlaw.py added as a license file.